website/docs/reference/commands/version.md

```diff
@@ -4,10 +4,20 @@ sidebar_label: "version"
 id: "version"
 ---
 
-The `--version` command-line flag returns information about the currently installed version of dbt Core or the dbt Cloud CLI.
+The `--version` command-line flag returns information about the currently installed version of dbt Core or the dbt Cloud CLI. This flag is not supported when invoking dbt in other dbt Cloud runtimes (for example, the IDE or scheduled runs).
 
-This flag is not supported when invoking dbt in other dbt Cloud runtimes (for example, the IDE or scheduled runs).
+- **dbt Core** &mdash; Returns the installed version of dbt-core and the adapters (which is one local bundle).
```

Comment:
I think with v 1.8 these are installed separately now and we've decoupled core install from adapters. See [this](https://docs-getdbt-hmoknlxj2-dbt-labs.vercel.app/docs/core/pip-install#installing-the-adapter) for more info and make sure to select v 1.8!
```suggestion
- **dbt Core** &mdash; Returns the installed version of dbt-core and the adapters (which is one local bundle).
```

Reply:
@runleonarun Good callout!
```suggestion
- **dbt Core** &mdash; Returns the installed version of dbt-core and all adapters. Starting with v1.8, the dbt-core and adapter minor versions may diverge.
```
That's also worth updating here:
- https://docs.getdbt.com/docs/dbt-versions/core#how-we-version-adapter-plugins

Reply:
thanks y'all!! 

since the following section **Version** links to "How dbt Core uses semantic versioning" which has the subsection "How we version adapter plugins" (which already talks about independent releases and minor version), i updated that bullet item mainly to remove "one bundle". as a best practice in user docs, we want to manage link-outs so it's not repetitive on such a short page

Reply:
> "How we version adapter plugins" (which already talks about independent releases and minor version)

Sorry - this was me remembering that we will need to update this page in advance of v1.8 final release. I just added a comment to this issue (I think related), but can open a new issue to track this follow-up work:
- https://github.com/dbt-labs/docs.getdbt.com/issues/5104#issuecomment-2035100934